lib/gcloud/resource_manager.rb

```diff
@@ -84,9 +84,13 @@ def self.resource_manager keyfile = nil, options = {}
   #
   # == Accessing the Service
   #
-  # The Resource Manager API *cannot* be enabled in the developer console, and
-  # can only be enabled by your account manager or a member of the Google Cloud
-  # sales team.
+  # Currently, the full functionality of the Resource Manager API is available
+  # only to whitelisted users. (Contact your account manager or a member of the
+  # Google Cloud sales team if you are interested in access.) The read-only methods
+  # {projects.list}[https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/list] and
```

Comment:
Certainly not yet. Let's let user feedback drive that.
I'll push an updated commit in a minute. I agree, better to point to the immediate API.